.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,53 @@
+
+name: Pull Request
+
+on:
+  pull_request:
+    types:
+      - opened
```

Comment:
question(possibly-blocking): does this catch the case when someone does `git push` again on a branch? I guess that `synchronized` should be added [link](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)

Reply:
From Github:

> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened.

we can get rid of `types` entirely

Reply:
actually now that I pushed a second revision, the check didn't run, so it's exactly as you said.
I will remove it and add a comment. Thank you